api/src/controllers/topics.js

```diff
@@ -41,6 +46,7 @@ export const createOrUpdateTopic = async (req, res) => {
 };
 
 const githubFunc = async (
```

Comment:
what does this do?

no js docs?

why is this in the same file if its a helper